### DISCUSSION TITLE
v2 in the name

### DISCUSSION THREAD

**Discussion by aahnik** [deepjyoti30/ytmdl-web-v2#20] [General]
Why `v2` in the name of the repo? I believe this project will stay for a long time, and many newer releases will be released. 

Just `ytmdl-web` sounds short and sweet. I am curious to know the reason behind your naming.

Never worry about renaming, because GitHub will automatically redirect people who come to the old URL of this repo.

**Comment by deepjyoti30** [comment]
@aahnik Actually before this version was released, the first version repo had the name `ytmdl-web`. I thought it would be fun to add the `-v2` since this is like the `2.0` of the web version.

Perhaps later, I will rename this to just `ytmdl-web`!

Also you can use [discussions](https://github.com/deepjyoti30/ytmdl-web-v2/discussions) for questions like this. Cheers!

**Comment by aahnik** [comment]
Yup, Discussions did not exist when I asked this. Thanks for creating. 

You may move this issue to a discussion.

![image](https://user-images.githubusercontent.com/66209958/119846730-047f9800-bf28-11eb-8d03-de507becf868.png)





> the first version repo had the name ytmdl-web

I don't understand this. That's why a version control system like git is used. To have all versions in one repo. Perhaps there were massive changes, so you thought to create a new repo all together.

> Perhaps later, I will rename this to just ytmdl-web!

Wow! that will a great decision.

**Reply by deepjyoti30** [reply]
@aahnik 

> I don't understand this. That's why a version control system like git is used.

Yes. You are correct but the second version was built from ground up. That's why I decided to go for a separate repo.
